package ProtoNotify;

message NotifyItemCPP
{
	required uint32 id		= 1;  //通知id
	required uint32 ts		= 2;   //时间
	required bytes content	= 3; //内容
}

message GetNotifyReq
{
	required uint32 id	= 1;
}

message GetNotifyResp
{
	required NotifyItemCPP  notify	= 1;  //通知
}